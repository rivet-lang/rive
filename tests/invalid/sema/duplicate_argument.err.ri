fn main() {}

fn my_fn(a: int, b: int) {} // OK

fn win(a: int, a: int) {} // FAIL
