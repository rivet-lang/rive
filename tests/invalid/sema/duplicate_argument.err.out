tests/invalid/sema/duplicate_argument.err.ri:5:16: error: duplicate argument `a`
      | 
    5 | fn win(a: int, a: int) {} // FAIL
      |                ^~~~~~            
      = note: inside function `win`
error: could not compile `duplicate_argument` module, aborting due to previous error